lib/components/base_component.dart

```diff
@@ -56,13 +56,12 @@ abstract class BaseComponent extends Component {
   @mustCallSuper
   @override
   void render(Canvas canvas) {
+    canvas.save();
     prepareCanvas(canvas);
 
     if (debugMode) {
       renderDebugMode(canvas);
     }
-
-    canvas.save();
```

Comment:
Maybe we should have an optional argument for `BaseComponent` whether children should be rendered in relation to the parent or not?